package.json

```diff
@@ -272,8 +271,7 @@
     "eslint": "^7.1.0",
     "//": "pretty-format contains typing only supported by TS 3.8+ remove when support in this repo is available",
     "@testing-library/dom": "7.22.3",
-    "copy-to-clipboard": "3.2.0",
-    "@storybook/react-docgen-typescript-plugin": "1.0.2-canary.f7e28c0.0"
```

Comment:
This resolution was to downgrade to `react-docgen-typescript` 1, since `react-docgen-typescript` 2 requires TS 4.3+. Now that we're on 4.3, we can remove the resolution.